roles/alexandria-v2/templates/environment.rb.j2

```diff
@@ -0,0 +1,7 @@
+# Load the Rails application.
+require File.expand_path('../application', __FILE__)
+
+Rails.env = ENV['RAILS_ENV'] || '{{ rails_env }}'
```

Comment:
I want to replace `environment.rb` with a `.template` in ADRL.  I've found `Rails.env = ENV['RAILS_ENV'] || '{{ rails_env }}'` really useful in keeping the environment consistent without having to remember to set `RAILS_ENV` for basic things like `rails console`.